Sponsor name added

mahikaw opened the pull request:
Fixes #1426

Changes: [Add here what changes were made in this issue and if possible provide links.]
Sponsor names have also been added in addition to image and sponsor type.
Screenshots for the change: 

![device-2017-05-08-225738](https://cloud.githubusercontent.com/assets/21010060/25816893/35104c04-3443-11e7-95f4-c62347d38aad.png)

the-dagger commented on android/app/src/main/res/layout/item_sponsor.xml:
```diff
@@ -18,6 +18,17 @@
+
+        <TextView
+            android:id="@+id/sponsor_name"
+            android:layout_width="79dp"
```
Why a fixed size?
I think this is the reason as to why the Sponsor name isn't properly centralized for most of the sponsors.

the-dagger commented on android/app/src/main/res/layout/item_sponsor_type.xml:
```diff
@@ -21,4 +21,5 @@
         android:textStyle="bold|italic"
         tools:text="Sponsor" />
 
+
```
Avoid adding empty lines

the-dagger commented on android/app/src/main/res/layout/item_sponsor.xml:
```diff
@@ -27,6 +38,8 @@
             android:textStyle="bold|italic"
             tools:text="Sponsor Type" />
+
+
```
Same.

iamareebjamal commented:
@mahikaw Also, please squash the commits

mahikaw commented:
@iamareebjamal I'm not sure how to squash on GitHub since I made the final changes like removing empty lines on GitHub only. Maybe, @the-dagger could squash them while merging. :)

free4murad commented:
@mahikaw 
Here read this
https://github.com/free4murad/Git-Guide

the-dagger commented:
@mahikaw 
I can squash them while merging, but it's a good practice to squash it beforehand.

https://ariejan.net/2011/07/05/git-squash-your-latests-commits-into-one/

mahikaw commented:
@the-dagger @free4murad I am aware off this and I have squashed commits that I changed locally, however when I edit on GitHub itself, and commit online, those commits don't appear in git log and so I am not sure how can I squash them. Could you help me with that?

the-dagger commented:
@mahikaw Could you check and see if the sponsor names are centralized now?

iamareebjamal commented:
@mahikaw You have to pull the topic branch locally first, then they will be visible for edits

arp95 commented:
@mahikaw 
apart from that can you please follow the commit guidelines.
Thanks

mahikaw commented:
@the-dagger I have incorporated the changes requested and squashed the commits. Here is the screenshot of the final changes. Please review. Thanks.

![device-2017-05-09-133818](https://cloud.githubusercontent.com/assets/21010060/25841456/d5acbdbc-34bd-11e7-8989-8788cd019255.png)

arp95 commented on android/app/src/main/java/org/fossasia/openevent/adapters/SponsorsListAdapter.java:
```diff
@@ -203,6 +204,9 @@ public int getItemViewType(int position) {
         @BindView(R.id.sponsor_type)
         TextView sponsorType;
 
+        @BindView(R.id.sponsor_name)
```
Please resolve the codacy issue.
Rest i think is fine.

the-dagger replied:
@arp95 
I think it's ok, codacy gives this issue with butterknife.
This isn't an issue as such.
